packaging/hoodie-hadoop-mr-bundle/src/main/java/com/uber/hoodie/hadoop/bundle/Main.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2017 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ */
+
+package com.uber.hoodie.hadoop.bundle;
+
+import com.google.common.reflect.ClassPath;
+import java.lang.String;
+
+import java.io.IOException;
+
+/**
+ * A simple main class to dump all classes loaded in current classpath
+ */
+public class Main {
```

Comment:
Could you please explain the changes here ? Also, could you explain the need for the same class everywhere ? Is there a way to avoid these repetitions ?
@bvaradar

Reply:
@n3nash : Sure,  This is a workaround for generating sources and javadoc jars for packaging modules. The maven plugins for generating javadoc and sources plugins do not generate corresponding jars if there is no source files.  I tried few things including setting sourceDirectory but nothing worked. 

Having dummy class in each of these packages solved the problem.  The 3 classes are like HelloWorld class which does not have anything to do with Hudi but is there to keep mvn javadocs/source plugin happy.

I have added this description to  each of these classes.